aws/resource_aws_rds_cluster_test.go

```diff
@@ -3518,3 +3546,100 @@ resource "aws_rds_cluster" "test" {
 }
 `, rName, enableHttpEndpoint)
 }
+
+func testAccAWSRDSClusterConfig_MultipleClustersInGlobalCluster(rNameGlobal, rNamePrimary, rNameSecondary string) string {
+	return composeConfig(
+		testAccMultipleRegionProviderConfig(2),
+		fmt.Sprintf(`
+data "aws_region" "current" {}
+
+data "aws_availability_zones" "alternate" {
+  provider = "awsalternate"
+  state = "available"
+
+  filter {
+    name   = "opt-in-status"
+    values = ["opt-in-not-required"]
+  }
+}
+
+resource "aws_rds_global_cluster" "test" {
+  global_cluster_identifier = "%[1]s"
+  engine                    = "aurora-mysql"
+  engine_version            = "5.7.mysql_aurora.2.07.1"
+}
+
+resource "aws_rds_cluster" "primary" {
+  cluster_identifier              = "%[2]s"
+  database_name                   = "mydb"
+  master_username                 = "foo"
+  master_password                 = "barbarbar"
+  skip_final_snapshot             = true
+  global_cluster_identifier       = aws_rds_global_cluster.test.id
+  engine                          = "aurora-mysql"
+  engine_version                  = "5.7.mysql_aurora.2.07.1"
```

Comment:
Nit: Might simplify maintenance down the road to reference these from the `aws_rds_global_cluster` resource directly.

```suggestion
  engine                          = aws_rds_global_cluster.test.engine
  engine_version                  = aws_rds_global_cluster.test.engine_version
```

Similarly with all the hardcoded ones below as well, e.g. switching the instance below to reference `aws_rds_cluster.primary.engine` etc. 👍

Reply:
yes totally agree! forgot these references make things so much easier 😅